app/views/layouts/application.html.haml

```diff
@@ -28,7 +28,7 @@
 
     = render partial: 'layouts/cookie_banner'
 
-    %header.govuk-header{"data-module" => "govuk-header", role: "banner"}
+    %header.govuk-header{"data-module" => "govuk-header", role: "banner", class: ENV['ENV'] }
```

Comment:
I am wondering whether we could expand this out a bit, so the class reads "app-environment-dev" (for example).  I assume this would be `class: "app-environment-" + ENV['ENV']` or something like that.  

This makes it more descriptive and less likely to clash with something else should a new class of "dev" or "uat" be added at some point in the future.  

I would have put this on the body tag, but here is okay too.

Reply:
See my comment about `ENV['ENV']` below - I suggest setting a new variable to the value and using it throughout.

Reply:
👍 have expanded class name to "app-environment-" as suggested.